docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -442,10 +302,129 @@ This file is picked up when the server starts as indicated by the following AUDI
+
+    For a Derby-related datasource definition, any bean property supported by the `EmbeddedXADataSource` class can be specified within the `connectionProperties` property group. For more information about the properties supported by the `EmbeddedXADataSource` class, and its super classes, see the [Apache Derby documentation](https://db.apache.org/derby/docs/10.13/publishedapi/org/apache/derby/jdbc/EmbeddedXADataSource.html).
+
+
+When the proxy datasource is obtained via a JNDI lookup by the JDBC persistence layer, and then its “getConnection()” method is called, the proxy datasource will use the current tenant-id and datastore-id to retrieve the corresponding configuration properties from the `fhir-server-config.json` file. The result will be the instantiation (or a successful cache lookup) of the appropriate XADataSource implementation class (according to the `type` field in the datastore configuration) and then a new connection will be obtained from it.
+
+The proxy datasource relies on the presence of the tenant-id and datastore-id information within the thread-local `FHIRRequestContext` information. In a simple configuration of the FHIR server that involves the use of the JDBC persistence layer implementation, the `FHIRRequestContext` information is obtained via request headers from the incoming REST API request. The tenant-id is obtained from the `X-FHIR-TENANT-ID` request header and the datastore-id is obtained from the `X-FHIR-DSID` request header<sup id="a3">[3](#f3)</sup>.
+
+#### 3.3.1.3 Supported databases
+##### 3.3.1.3.1 Embedded Derby (default)
+If you are using the `ibmcom/ibm-fhir-server` docker image, you can ask the entrypoint script to create (bootstrap) the database and the schema and tables during startup by setting the `DB_BOOTSTRAP` environment variable to `true`.
+
+This database bootstrap step is only supported for Embedded Derby and will only bootstrap the default datastore of the default tenant (the default for requests with no tenant or datastore headers).
+
+```
+<server>
+    <!-- ============================================================== -->
+    <!-- This datasource aligns with the Apache Derby database that is  -->
+    <!-- created by the IBM FHIR Server's DB_BOOTSTRAP process.         -->
+    <!-- ============================================================== -->
+    <dataSource id="fhirDefaultDefault" jndiName="jdbc/fhir_default_default" type="javax.sql.XADataSource" statementCacheSize="200" syncQueryTimeoutWithTransactionTimeout="true">
+        <jdbcDriver javax.sql.XADataSource="org.apache.derby.jdbc.EmbeddedXADataSource" libraryRef="fhirSharedLib"/>
+        <properties.derby.embedded createDatabase="create" databaseName="derby/fhirDB"/>
+        <connectionManager maxPoolSize="50" minPoolSize="10"/>
+    </dataSource>
+</server>
+```
+
+##### 3.3.1.3.2 Db2
+If you configure the FHIR server to use an IBM Db2 database, you must:
+
+1. create the database if it doesn't already exist
```

Comment:
For each of these steps, capitalize first word and add period at the end (for consistency).